core/scripts/cli/sponsor/redeem.js

```diff
@@ -12,15 +12,7 @@ const redeem = async (web3, artifacts, emp) => {
   const token = await SyntheticToken.at(tokenAddress);
   const walletTokens = await token.balanceOf(sponsorAddress);
 
-  console.log("You have:");
-  console.log(
-    "Position: " +
-      fromWei(collateral) +
-      " WETH backing " +
```

Comment:
Why remove the messaging about collateral? I thought it was helpful to know how much collateral I had

Reply:
AH I see you show it in showMarketDetails

Reply:
Yep, just moved it all to one place.